2-numpy/manual.ipynb

```diff
@@ -265,9 +265,9 @@
     "Something similar can be constructed with Python lists, where the `list` of rows each contain a column `list`.\n",
     "These are then accessed by first indexing the row and then the column like `diffs[i][j]`. \n",
     "\n",
-    "### Masking\n",
+    "### Boolean indexing\n",
```

Comment:
```suggestion
    "### Boolean array indexing\n",
```
The word "array" is important. I am highlighting it only here, but this also needs to be updated elsewhere.

Reply:
Resolved in latest commit.